mlflow/server/job_api.py

```diff
@@ -0,0 +1,69 @@
+"""
+Internal job APIs for UI invocation
+"""
+import os
+from typing import Any
+
+from fastapi import APIRouter, HTTPException, status as http_status
+from pydantic import BaseModel
+
+
+job_api_router = APIRouter(prefix="/ajax-api/3.0/job", tags=["Job"])
```

Comment:
```suggestion
job_api_router = APIRouter(prefix="/ajax-api/3.0/jobs", tags=["Job"])
```

let's make this plural.